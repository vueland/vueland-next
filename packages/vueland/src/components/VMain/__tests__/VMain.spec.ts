import { mount } from '@vue/test-utils'
import { VMain } from '../index'

describe('VMain', () => {
  it('should mount component and match snapshot', () => {
    const wrapper = mount(VMain)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
