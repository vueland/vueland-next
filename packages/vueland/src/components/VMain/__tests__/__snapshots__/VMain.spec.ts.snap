// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VMain should mount component and match snapshot 1`] = `
<div class="v-main">
</div>
`;
